### ISSUE TITLE
Add alt text for figure images

### ISSUE THREAD

**Issue by geealbers** [thegetty/artists-things#48]
To increase the accessibility of the book, alt descriptions should be added to all figure images in the book. Follow [Getty Publications' Alt Text Guidelines](https://getty-quire-guide.netlify.app/guides/alt-text/) when crafting the descriptions.

Work off of the `revisions` branch, and be sure to note the Node and Quire CLI versions the project uses.

Please submit a chapter's worth as a pull request to start. After which you can proceed and your own pace and submit a PR whenever you feel it's ready.
